srcpkgs/beancount/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'beancount'
+pkgname=beancount
+version=2.1.3
+revision=1
+build_style=python3-module
+pycompile_module="beancount"
+hostmakedepends="python3-devel python3-setuptools"
+makedepends="python3-devel"
+depends="python3-pytest python3-dateutil python3-ply python3-bottle python3-lxml
+ python3-magic python3-BeautifulSoup4 python3-requests python3-chardet
+ python3-google-api-python-client"
+short_desc="A double-entry accounting system that uses text files as input"
```

Comment:
short_desc should not start with an article.

Reply:
Fixed.